### PR TITLE
Release process document

### PR DESCRIPTION
**Pull request by shana**
Start documenting the release process, add template for release notes.

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+#### The long explanation
+
+Every week:
+
+1. Create a release branch from master with the name `release/[Major].[Minor]`, where `[Major].[Minor]` are the current major and minor parts of the version set on master.
+1. Bump the version on master
+1. Create a draft release with the release notes, following the format below
+1. Upload a build created from the release branch
+1. QA tests the release and logs any issues found in it. Issues are fixed on the next release unless they are related to the issues tjat are reported as fixed in the current release, and they are shipblockers.
```
Comment (StanleyGoldman) [inline docs/release-process.md]: `s/tjat/that`

File: docs/release-process.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Template for release notes
+
+The https://github.com/github-for-unity/unity-release-notes node project generates draft release notes by outputting all issues labeled `Feature`, `Enhancement` and `Bug` closed since the last release in the correct format for release notes, and also outputs all PRs closed since the last release, in case some issues might not have been correctly labeled and/or created.
+
+The markdown format for release notes is below. This is what the project above generates (more or less).
+
+```
+# Release notes
+
+[any special notes for the release go here, if needed]
```
Comment (StanleyGoldman) [inline docs/release-process.md]: Special notes... or fanfare?

Comment (shana) [reply docs/release-process.md]: Both?

**Comment by meaghanlewis** [comment]
One thing about organization though. Could this be moved under the folder `/process` and a link be added to the readme along with the other processes we follow.

File: docs/release-process.md (hunk 2 above)
Comment (StanleyGoldman) [reply docs/release-process.md]: Lol, I just wanted to be sure fanfare wasn't allowed. Depending on intended use...

Comment (StanleyGoldman) [reply docs/release-process.md]: Like the bit of html with an image like we've been doing lately

**Pull request merged**